.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
       - id: end-of-file-fixer
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.5.1
+    rev: v4.0.0-alpha.8
```

Comment:
Upgrading here is fine, but do you really want to depend on an alpha version? Plus, according to the README of the action, this is no longer maintained. Meaning, it will never reach a stable v4. I'd prefer to keep a stable v3 here.

Reply:
This line was automatically updated by running `pre_commit autoupdate`. It is a little surprising that an alpha version was pulled in, but the maintainer of that repo evidently has it listed as the latest tag and pre-commit must not care about pulling in only tags that follow semantic versioning reflecting a stable tag. We do technically use this alpha version in the Slicer pre-commit config (see [here](https://github.com/Slicer/Slicer/blob/1af67ed6f1411d9dac9f8d7fe246469bb0556569/.pre-commit-config.yaml#L33)), but I have changed it back to the latest stable. Ultimately I will probably remove it from the pre-commit configuration entirely and switch to a newer maintained equivalent.